_pages/meta-score.md

```diff
@@ -0,0 +1,168 @@
+---
+title: HippoPenny Score
+layout: scores
+permalink: /meta-score/
+show_excerpts: true
+sort_order: reverse
+sort_by: date
```

Comment:
what is the order of posts in the same year, same day? I think you want by hour, minutes (convert datetime to unix timestamp, and sort them)

Reply:
I will change